Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+RUN apt-get update
+
+RUN mkdir -p /home/Bassa
+RUN apt-get install git
+RUN git clone https://github.com/scorelab/Bassa.git /home/Bassa
```

Comment:
Hi @kmehant, we don't need cloning the repo with git here. We can simply copy the files from the local repo. 

You can refer: https://docs.docker.com/engine/reference/builder/#copy

Reply:
@agentmilindu I have a doubt here, can i say

```
WORKDIR /home/Bassa
COPY Bassa WORKDIR 
```

Is this valid?

Reply:
You can say `COPY . /home/Bassa `

http://blog.cloud66.com/how-to-get-code-into-a-docker-container/#option3usingtheaddorcopycommand